common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1311,6 +1311,10 @@
   "screens.wynntils.map.help.description8": "Right click to center on player.",
   "screens.wynntils.map.help.description9": "Shift-Right click to center on the compass marker.",
   "screens.wynntils.map.help.name": "Quick Guide",
+  "screens.wynntils.map.manager.description1": "List, Delete or Create",
+  "screens.wynntils.map.manager.description2": " all your preview set",
```

Comment:
This does not really make sense?

Reply:
I just copied it from legacy originally and noticed that too, just forgot to change it to something better